libs/barista-components/stacked-series-chart/src/stacked-series-chart.ts

```diff
@@ -487,13 +507,44 @@ export class DtStackedSeriesChart implements OnDestroy, OnInit {
    * Handles the mouseLeave on a series slice.
    * Dismisses the overlay if there is one defined.
    */
-  _handleOnSeriesMouseLeave(): void {
+  _handleOnSeriesMouseLeave(slice: DtStackedSeriesChartTooltipData): void {
+    this.hoverEnd.emit(this._trackStackHoverEvents(slice));
     if (this._overlayRef) {
       this._overlayRef.dismiss();
       this._overlayRef = null;
     }
   }
 
+  _handleOnLegendMouseLeave(legend: DtStackedSeriesChartLegend): void {
+    this.hoverEnd.emit(this._trackLegendHoverEvents(legend));
+  }
+
+  private _trackStackHoverEvents(
+    slice: DtStackedSeriesChartTooltipData,
+  ): DtStackedSeriesStackHoverData {
+    return {
+      value: slice.valueRelative,
+      stackName: slice.origin.label,
+      seriesName: slice.seriesOrigin.label,
+      color: slice.color,
+      selected: slice.selected,
+      visible: slice.visible,
+      hoveredIn: 'stack',
+    };
+  }
+
+  /** @internal returns an object with output data type for on hover events */
```

Comment:
You don't need to mark private members as internal. Only public ones that you would not want to have accessible to the consumer. 
Sometimes you need to make field public, because you need them in the template, like the _handlemouse* from above. 
```suggestion
  /** returns an object with output data type for on hover events */
```